cub/benchmarks/bench/scan/applications/log-cdf-from-log-pdfs.cu

```diff
@@ -0,0 +1,180 @@
+  T norm;
+  cuda::std::size_t n;
+
+  T __host__ __device__ operator()(cuda::std::size_t i) const
+  {
+    return -mu * static_cast<T>(n - i) + norm;
+  }
+};
+
+template <typename T>
+[[nodiscard]] bool validate(const thrust::device_vector<T>& output)
+{
+  thrust::host_vector<T> h_output(output);
+  auto elements = h_output.size();
+  // test is designed so that last element of prefix scan sequence should be close to log(1.0) == 0.0
+  bool check = cuda::std::abs(h_output[elements - 1])
+             < cuda::std::sqrt(static_cast<T>(1 + elements)) * cuda::std::numeric_limits<T>::epsilon();
+
+  return check;
+}
+
+}; // namespace impl
+
+template <typename FloatingPointT, typename OffsetT>
+static void inclusive_scan(nvbench::state& state, nvbench::type_list<FloatingPointT, OffsetT>)
+{
+  using wrapped_init_t = cub::NullType;
+  using value_t        = FloatingPointT;
+  using input_t        = const value_t*;
+  using output_t       = value_t*;
+  using offset_t       = cub::detail::choose_offset_t<OffsetT>;
+  using op_t           = impl::LogAddPlus;
+  using accum_t        = value_t;
+
+#if !TUNE_BASE
+  using policy_t = policy_hub_t<accum_t>;
+  using dispatch_t =
+    cub::DispatchScan<input_t, output_t, op_t, wrapped_init_t, offset_t, accum_t, cub::ForceInclusive::No, policy_t>;
+#else
+  using dispatch_t =
+    cub::DispatchScan<input_t, output_t, op_t, wrapped_init_t, offset_t, accum_t, cub::ForceInclusive::No>;
+#endif
+
+  const auto elements = static_cast<std::size_t>(state.get_int64("Elements{io}"));
+  auto mu             = static_cast<value_t>(state.get_float64("Mu{io}"));
+
+  auto norm = cuda::std::log1p(-cuda::std::exp(-mu)) - cuda::std::log1p(-cuda::std::exp(-mu * elements));
+
+  thrust::device_vector<value_t> input(elements, thrust::no_init);
+
+  cudaStream_t bench_stream = state.get_cuda_stream();
+
+  cub::DeviceTransform::Transform(
+    cuda::std::make_tuple(cuda::counting_iterator(cuda::std::size_t{0})),
+    input.begin(),
+    elements,
+    impl::LogPDBuilder<value_t>{mu, norm, elements},
+    bench_stream);
+
+  thrust::device_vector<value_t> output(elements, thrust::no_init);
+
+  input_t d_input   = thrust::raw_pointer_cast(input.data());
+  output_t d_output = thrust::raw_pointer_cast(output.data());
+
+  state.add_element_count(elements);
+  state.add_global_memory_reads<value_t>(elements, "Size");
+  state.add_global_memory_writes<value_t>(elements);
+
+  size_t tmp_size;
+  dispatch_t::Dispatch(nullptr, tmp_size, d_input, d_output, op_t{}, wrapped_init_t{}, input.size(), bench_stream);
+
+  thrust::device_vector<nvbench::uint8_t> tmp(tmp_size, thrust::no_init);
+  nvbench::uint8_t* d_tmp = thrust::raw_pointer_cast(tmp.data());
+
+  state.exec(nvbench::exec_tag::gpu | nvbench::exec_tag::no_batch, [&](nvbench::launch& launch) {
+    dispatch_t::Dispatch(
+      d_tmp, tmp_size, d_input, d_output, op_t{}, wrapped_init_t{}, input.size(), launch.get_stream());
+  });
+
+  cudaStreamSynchronize(bench_stream);
```

Comment:
Question: was this left in by mistake? None of the other benchmarks synchronize in this way. Additionally, I believe you should be using `launch.get_stream()` (if you were to use it).